src/js/common/schemaform/save-load-actions.js

```diff
@@ -111,12 +109,16 @@ export function saveInProgressForm(formId, version, returnUrl, formData) {
     }),
     formData: JSON.stringify(formData)
   });
-  return dispatch => {
+  return (dispatch, getState) => {
```

Comment:
Adding to my TODO list: Learn more about redux thunk...

I didn't know you could also have `getState`. Handy!